data/campaigns/The_South_Guard/scenarios/03_A_Desparate_Errand.cfg

```diff
@@ -210,28 +210,38 @@
 
         [message]
             speaker=narrator
-            message= _ "Deoran and his men came to the eastern border of the Aethenwood..."
+            message= _ "Deoran and his men came to the eastern border of the Aethenwood, where the towering trees of the elfin forest loomed in the distance."
             image=wesnoth-icon.png
         [/message]
 
         [message]
             speaker=Deoran
-            message= _ "Look, men! There is the forest of the Elves to the northwest. Let us hasten across the river and the hills."
+            message= _ "Look to the north! There lies the forest of the elves. Let us hasten across the river and the hills."
         [/message]
 
         [message]
             speaker=Minister Hylas
-            message= _ "Be careful, Deoran. No man has set foot in the elvish woods for many a year. We do not know how they will receive us. Sir Loris thought that it was some trick of theirs trying to bring him into the woods..."
+            message= _ "I must advise caution, Sir Deoran. No man has set foot in the elvish woods for many a year. They have never been friendly toward us men, and we do not know how they will receive us now."
         [/message]
 
         [message]
             speaker=Deoran
-            message= _ "But if we do not discover the source of these undead warriors, the whole province shall surely be swallowed up by them, and then their bandit masters will fare no better. We cannot allow it!"
+            message= _ "We cannot fear and shun everything we do not understand! Sir Loris may have distrusted the elves, but he is gone now, and I daresay as a result of his own folly! We do not possess the strength to fight these undead, nor the means to discover their source, and so we can only look to these elves for help, wary of men though they may be."
```

Comment:
After the change above, do we have dialog establishing Loris as wary of the elves? "as so" is a run-on warning flag— a simple improvement might be "We do not possess the strength to fight these undead, nor the means to discover their source; we can only look to these elves for help, wary of men though they may be."

Reply:
We do have evidence from scenario 2, where this conversation occurs:

        [message]
            speaker=Minister Hylas
            message= _ "Just before the bandit raids began, an ambassador came to Sir Loris from the elves of the Aethenwood. He asked that a knight be sent to discuss a new threat to Wesnoth."
        [/message]

        [message]
            speaker=Deoran
            message= _ "And did Sir Loris go to the elves?"
        [/message]

        [message]
            speaker=Minister Hylas
            message= _ "No. He suspected the elves of some trickery and made no reply to them. Perhaps they know the origin of these dead warriors we now face."
        [/message]

I put in the other change.